ts/db_test.go

```diff
@@ -56,6 +57,7 @@ func newTestModel(t *testing.T) *testModel {
 	return &testModel{
 		t:                t,
 		modelData:        make(map[string]*proto.Value),
+		seenSources:      make(map[string]bool),
```

Comment:
If this is really just a set, the canonical way to do it is `map[string]struct{}`